cmd/appgw-ingress/utils.go

```diff
@@ -48,6 +48,12 @@ func validateNamespaces(namespaces []string, kubeClient *kubernetes.Clientset) e
 func getNamespacesToWatch(namespaceEnvVar string) []string {
 	// Returning an empty array effectively switches Ingress Controller
 	// in a mode of observing all accessible namespaces.
+	if len(namespaceEnvVar) != 0 {
```

Comment:
We already have print statement for namespaces in the main.go file.
https://github.com/Azure/application-gateway-kubernetes-ingress/blob/ca2f88cec85dca87d072f68662fa556ddb9ced1a/cmd/appgw-ingress/main.go#L167

Reply:
Thanks, removed